modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/VolatilePageMemoryMvPartitionStorage.java

```diff
@@ -34,20 +34,25 @@ public class VolatilePageMemoryMvPartitionStorage extends AbstractPageMemoryMvPa
     /**
      * Constructor.
      *
-     * @param partId Partition id.
-     * @param tableView Table configuration.
-     * @param pageMemory Page memory.
-     * @param rowVersionFreeList Free list for {@link RowVersion}.
+     * @param tableStorage Table storage instance.
+     * @param partitionId Partition id.
      * @param versionChainTree Table tree for {@link VersionChain}.
+     * @param indexMetaTree Tree that contains SQL indexes.
      */
```

Comment:
```suggestion
     * @param indexMetaTree Tree that contains SQL indexes' metadata.
```
Or something like that?